src/components/chart/Chart.component.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+    CartesianGrid,
+    Line,
+    LineChart,
+    ResponsiveContainer,
+    Tooltip,
+    XAxis,
+    YAxis,
+} from 'recharts';
+
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+import { getChartStyles } from './Chart.styles';
+import { ChartSectionProps } from './Chart.types';
+import { ChartTooltip } from './ChartTooltip.component';
+
+export const Chart = ({ data }: ChartSectionProps) => {
+    const theme = useTheme();
+    const styles = getChartStyles(theme);
+    const isSmall = useMediaQuery(theme.breakpoints.down('md'));
+    const chartStyles = getChartStyles(theme);
+    const chartHeight = isSmall ? 407 : 446;
+
+    return (
+        <ResponsiveContainer
+            width="100%"
+            height={chartHeight}
+            aria-label="Line chart showing weekly sales data"
+        >
+            <LineChart
+                data={data}
+                margin={{ top: 20, right: 30, left: 0, bottom: 10 }}
+                aria-describedby="chart-description"
+            >
+                ``
+                <CartesianGrid {...styles.grid} />
+                <XAxis
+                    {...chartStyles.xAxis(isSmall)}
+                    dataKey="date"
```

Comment:
this component is not really reusable if these axes are to be hardcoded. These things should be props